vscode-lean4/src/leanclient.ts

```diff
@@ -240,17 +240,8 @@ export class LeanClient implements Disposable {
         } as any, null);
 
         // HACK
```

Comment:
```suggestion
        // Reveal the standard error output channel when the server prints something to stderr.
        // The vscode-languageclient library already takes care of writing it to the output channel.
```
Let's improve this comment here so that the next person knows what's up.